react/typings/events.d.ts

```diff
@@ -140,21 +141,112 @@ export interface ProductImpressionData extends EventData {
   list: string
 }
 
+export interface CartData extends EventData {
+  event: 'cart'
+  eventName: 'vtex:cart'
+  orderForm: OrderForm
```

Comment:
Can you also add this new event to 
https://github.com/vtex-apps/pixel-app-template

Reply:
sure

Reply:
done in vtex-apps/pixel-app-template#13